backend/app/exporters/serializers/mods.rb

```diff
@@ -81,7 +81,7 @@ def serialize_mods_inner(mods, xml)
           xml.temporal term
         when 'uniform_title'
           xml.titleInfo term
-        when 'genre_form', 'style_period'
+        when 'genre_form', 'style_period', 'technique'
```

Comment:
@avatar382 @cdibella From spreadsheet line 231, should this also include 'function'?

Reply:
Yes, function should be in there too - all three map to the mods tag genre.